src/poliastro/bodies.py

```diff
@@ -89,6 +144,15 @@ class _Venus(_Body):
     symbol = u"\u2640"
     R = constants.R_venus
 
+    @staticmethod
+    def _rot_elements_at_epoch(T, d):
+
```

Comment:
I think you can remove the white lines under the method definition (present only in some of them)